packages/verifiable-credential/README.md

```diff
@@ -0,0 +1,40 @@
+# VerifiableCredential DataModel
+
+DataModel implementation of the **VerifiableCredential** schema and definition.
+
+## Installation
+
+```sh
+npm install -D @datamodels/verifiable-credential
+```
+
+## Rationale
+
+A VerifiableCredential is fact-checking review/attestation of claims made (or reported) in a [ClaimedCredential](../claimed-credential/). This tries to follow the [W3C VC datamodel](https://www.w3.org/TR/vc-data-model), but adds alternative solidity compliant syntax for some conflicting property names.
+
+Tiles created using this schema can then be linked to a DID issued credentials on the [CredentialRegistry](../credential-registry/).
+
+## Schemas
+
+### [VerifiableCredential](./schemas/VerifiableCredential.json)
+
+The VerifiableCredential schema defines the format of a document that contains the properties listed below. Properties not defined in the schema _cannot_ be included in the VerifiableCredential.
+
+| Property          | Description                                                 | Value                                         | Max Size | Required | Example                                                   |
+| ----------------- | ----------------------------------------------------------- | --------------------------------------------- | -------- | -------- | --------------------------------------------------------- |
+| @context          | See https://www.w3.org/TR/vc-data-model/#contexts           | array of string                               |          | false    |                                                           |
+| \_context         | Provided for solidity syntax compatibility                  | comma separated string                        |          | false    |                                                           |
```

Comment:
Any details about the need for solidity compatibility here please? If it's derived data from other fields, shouldn't it be a runtime concern to do the conversion?

Reply:
You're right, I removed the solidity specific part from the datamodel